start_redact.sh

```diff
@@ -3,7 +3,8 @@ set -o errexit
 set -a
 installation_dir=${INSTALLATION_DIR}
 if [ -z $installation_dir ]; then
-    installation_dir="."
+    installation_dir=$(realpath ".")
+    export INSTALLATION_DIR=${installation_dir}
```

Comment:
why does this need to be exported?

Reply:
The `docker-compose.yaml` requires the `INSTALLATION_DIR` to be set as an env variable. The pwd solution didn't work.

Reply:
Thanks for clarifying